src/App.tsx

```diff
@@ -17,7 +17,7 @@ function App() {
             <Suspense fallback={<Skeleton />}>
               <Routes>
                 <Route path="/" element={<LandingPage />} />
-                <Route path="/dashboard/*" element={<DashRoutes />} />
+                <Route path="/*" element={<DashRoutes />} />
```

Comment:
This don't need to be changed actually. It will lead to this error

![image](https://user-images.githubusercontent.com/28519395/188901738-4ee4df5b-036b-4511-8b57-2035cddb76a3.png)

And I see no any reason why we would want to change that routing since they share no similarity with the landing page